javascript/implementation/CommandLineInterace.ts

```diff
@@ -38,12 +38,23 @@ export class CommandLineInterface {
         'token {key}' in their websocket subprotocol list.
         Otherwise, just accept all connections with the gink subprotocol.
         */
-        const authKey = process.env["GINK_AUTH_KEY"];
+        let authKey = process.env["GINK_AUTH_KEY"];
+        if (authKey && !authKey.toLowerCase().startsWith("token "))
+            throw new Error("make sure GINK_AUTH_KEY begins with 'token '");
+        // This is different than GINK_AUTH_TOKEN, which is what
```

Comment:
Is it confusing to use GINK_AUTH_TOKEN as a separate environment variable for the client? I thought it would be best to be certain if you were setting a key for a server or a token to connect with.

Reply:
yeah it's fine for now.